src/components/NFTEditionSelectItem.vue

```diff
@@ -110,6 +173,29 @@ export default {
         'transition-colors',
       ];
     },
+    discountInfo() {
+      const originalPrice = this.defaultPrice;
+      const { discountedPrice } = this;
+      if (originalPrice <= discountedPrice) {
+        return undefined;
+      }
+
+      const discountAmount = originalPrice - discountedPrice;
+      const discountPercentage = Math.ceil(
+        (discountAmount / originalPrice) * 100
+      );
+
+      return {
+        originalPriceLabel:
+          this.currency === 'HKD'
+            ? formatNumberWithUnit(
+                Number((originalPrice * USD_TO_HKD_RATIO).toFixed(1)),
```

Comment:
I think the server side logic is `Math.round((usdPriceInDecimal * USD_TO_HKD_RATIO) / 10) * 10;`, will this cause some difference in UI?

Reply:
Turns out it is used in other existing part, maybe just refactor this as function for everyone to use (and myabe improve later)